src/pages/CreateDesign/components/DropDown/DropDownOption/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { MenuItem, MenuItemProps } from '@material-ui/core';
+import React from 'react';
+
+interface Props {
+  onSelect?(value: number): void;
+  onClick?(value: number): void;
+
+  active: boolean;
+  disabled?: boolean;
+  value: number;
+  key: string;
+  children: React.ReactNode;
+}
+
+export const DropdownOption = ({
+  onSelect,
+  onClick,
+  active,
+  disabled,
+  value,
+  key,
+  children,
+}: Props): React.ReactElement => {
+  const handleClick: MenuItemProps['onClick'] = (event): void => {
+    if (!disabled) {
+      if (onSelect) {
+        onSelect?.(value);
+      }
+      if (onClick) {
+        event.stopPropagation();
+        onClick?.(value);
+      }
```

Comment:
```suggestion
      onSelect?.(value);
      if (onClick) {
        event.stopPropagation();
        onClick(value);
      }
```

p3: 이렇게 쓸 수 있지 않나요?!